.gitignore

```diff
@@ -15,4 +15,8 @@ build/
 [Oo]bj/
 [Rr]elease/
 _[Rr]e[Ss]harper.*/
-*.tss
\ No newline at end of file
+*.tss
```

Comment:
This is a more accurate representation

![image](https://cloud.githubusercontent.com/assets/50543/13569999/c7b17494-e46b-11e5-97e3-239dd8630d9b.png)